File: data/src/main/java/com/github/gmetal/data/entity/NetworkEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class NetworkEntity {

    @JsonField(name = "id")
    public Integer id;
    @JsonField(name = "name")
    public String name;
}

File: data/src/main/java/com/github/gmetal/data/entity/NetworkEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class NetworkEntity {

    @SerialName("id")
    var id: Int? = null
    @SerialName("name")
    var name: String? = null
}
